cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/BatchElasticsearchSink.java

```diff
@@ -36,7 +36,7 @@
 /**
  * A {@link BatchSink} that writes data to a Elasticsearch.
```

Comment:
Possibly. Perhaps instead: "to an Elasticsearch server." Is there only one such server, or is it specified somewhere? Certainly not "a Elasticsearch".